src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -159,7 +159,12 @@ public void teleportHome(@NotNull Player player, @NotNull Island island, int del
      */
     private void teleportHome(@NotNull Player player, @NotNull Island island) {
         player.setFallDistance(0);
-        PaperLib.teleportAsync(player, LocationUtils.getSafeLocation(island.getHome(), island), PlayerTeleportEvent.TeleportCause.PLUGIN);
+        if (PaperLib.isSpigot()) {
```

Comment:
Dont think this is needed since PaperLib should fall back onto spigot if paper isnt present automatically, also PaperLib.isSpigot will always return true since Paper is a fork of spigot so even if its running paper isSpigot will return true

Reply:
So its for checking is craftbukkit or spigot

Reply:
@return True for Spigot <em>and</em> Paper environments
yes my bad